Ginger/Ginger/AutomatePageLib/AddActionMenu/SharedRepositoryLib/ActivitiesRepositoryPage.xaml.cs

```diff
@@ -94,16 +98,20 @@ private void SetActivitiesRepositoryGridView()
 
             xActivitiesRepositoryGrid.btnRefresh.Visibility = Visibility.Collapsed;
             //grdActivitiesRepository.btnRefresh.AddHandler(Button.ClickEvent, new RoutedEventHandler(RefreshGridActivities));      
-            if (mContext != null && mContext.BusinessFlow!=null)
+            if (!mSRGroupRequest)
```

Comment:
all page modifications to allow using list for selection should have been done via eViewMode enum like we have on other items pages and not based on 'mSRGroupRequest' because we don't want to pass parameter per view type we will have

Reply:
Done !